users/migrations/0028_alter_user_username.py

```diff
@@ -0,0 +1,17 @@
+# Generated by Django 4.2.20 on 2025-04-29 13:08
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
```

Comment:
The migrations need to swap order so you widen the column and then populate the data over without truncating it.